apps/req.c

```diff
@@ -668,9 +668,9 @@ int req_main(int argc, char **argv)
             X509V3_set_nconf(&ext_ctx, req_conf);
 
             /* Add extensions */
-            if (req_exts && !X509V3_EXT_REQ_add_nconf(req_conf,
-                                                      &ext_ctx, req_exts,
-                                                      req)) {
+            if (req_exts != NULL && !X509V3_EXT_REQ_add_nconf(req_conf,
+                                                              &ext_ctx, req_exts,
```

Comment:
put the second part of the ```&&``` test on a new line, then you can un-indent the args to the funtion call.